docs/content/documentation/systems/Distributions/stochastic_tools/UniformDistribution.md

```diff
@@ -0,0 +1,18 @@
+# UniformDistribution
+!description /Distributions/UniformDistribution
+
+##Description
+The uniform distribution is a probability distribution that has constant probability.
+This is a continuous uniform distribution with the probability density function:
+f(x) = 1/(upper_bound - lower_bound) if lower_bound < x < upper_bound else zero.
```

Comment:
You can and should use latex, just do the following:

\begin{equation}
...
\end{equation}